testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -352,46 +426,122 @@ def test_atomgroup_to_atomgroup(self, u):
         atm = u.atoms.atoms
         assert len(atm) == 125
         assert isinstance(atm, groups.AtomGroup)
+        assert atm is u.atoms
```

Comment:
@orbeckst You required a test such as `assert ag.atoms is ag`. This line is equivalent.

Reply:
ok, will change my commit

Reply:
... and removed test_copy() as you have also covered everything